src/main/groovy/org/pih/warehouse/inventory/StockMovementStatusCode.groovy

```diff
@@ -32,6 +32,9 @@ enum StockMovementStatusCode {
     CANCELED(10),
     REJECTED(10, REJECTED),
 
+    // Status on the requestor's dashboard representing verifying, checked, picking and picked
+    IN_PROGRESS(3, PENDING),
```

Comment:
I wonder if this should not be the display status on its own (below the `PENDING`), without sort order, and should not take `PENDING` as its own displayStatusCode :thinking:

Reply:
> Is this getDisplayStatus() only used on the requestor's dashboard?
yes, it was introduced for mapping approval statuses for the requestor's dashboard